docs/src/swagger-output.json

```diff
@@ -2688,79 +3042,71 @@
             "schema": {
               "type": "object",
               "properties": {
-                "aplicationId": {
+                "type": {
                   "type": "string"
                 },
-                "essayId": {
-                  "type": "string"
+                "required": {
+                  "type": "boolean"
                 },
-                "hexathon": {
+                "ref": {
                   "type": "string"
                 },
-                "gradingGroup": {
-                  "type": "string",
-                  "enum": ["generalGroup", "emergingGroup"]
-                },
-                "score": {
-                  "type": "integer"
-                },
-                "isCalibrationQuestion": {
+                "index": {
                   "type": "boolean"
                 }
               }
             }
           }
         ],
         "responses": {
-          "201": {
-            "description": "successful operation"
-          }
-        }
-      }
-    },
-    "/grading/actions/skip-question": {
-      "post": {
-        "tags": ["registration"],
-        "summary": "Skips a question to grade",
-        "description": "",
-        "produces": ["application/json"],
-        "parameters": [
-          {
-            "name": "body",
-            "in": "body",
-            "description": "Request body",
-            "required": true,
+          "200": {
+            "description": "successful operation",
             "schema": {
-              "type": "object",
-              "properties": {
-                "hexathon": {
-                  "type": "string"
+              "type": "array",
+              "items": {
+                "properties": {
+                  "type": {
+                    "type": "string"
+                  },
+                  "required": {
+                    "type": "boolean"
+                  },
+                  "ref": {
+                    "type": "string"
+                  },
+                  "index": {
+                    "type": "boolean"
+                  }
```

Comment:
This part looks straight off of the model for tag. Instead, like for sponsor visit, let's have this as a separate definition. Also the properties aren't completely accurate, go over other examples in the docs to get an idea.